generative_ai/embeddings/document_retrieval_example.py

```diff
@@ -19,27 +19,41 @@
 
 
 def embed_text() -> list[list[float]]:
-    """Embeds texts with a pre-trained, foundational model.
```

Comment:
This comment is misleading (we do not "embed" texts anywhere, we calculate vector embeddings for them).